Makefile

```diff
@@ -0,0 +1,39 @@
+MAKEFLAGS += -j2
+default: help
+
+help:
+	@echo "Build Container"
+	@echo
+	@echo "  - make build"
+	@echo
+	@echo "Install ramalama"
+	@echo
+	@echo "  - make install"
+	@echo
+	@echo "Test ramalama"
+	@echo
+	@echo "  - make test"
+	@echo
+	@echo "Clean the repository"
+	@echo
+	@echo "  - make test"
+	@echo
+
+#
+# Create instructlab AI container images
```

Comment:
Planning on putting us under the instructlab banner? :smile: Trying to read the tea leaves on this comment

Reply:
oops copied from instructlab.